mlrun/feature_store/api.py

```diff
@@ -405,7 +417,7 @@ def ingest(
         raise mlrun.errors.MLRunInvalidArgumentError(
             "feature set and source must be specified"
         )
-
+    validate(featureset=featureset)
```

Comment:
function name is too generic, use something like validate_graph_steps()

Reply:
solved